### DISCUSSION TITLE
Paket Berbayar dan Perubahan Quota

### DISCUSSION THREAD

**Discussion by rioastamal** [kirimwa-id/forum#166] [Pengumuman]
**Update 18 Januari 2024**
- Paket berlangganan sudah dapat dipesan, untuk sementara hanya melalui email di rio@kirimwa.id

**Update 05 April 2023**
- Paket berlangganan terbaru diundur menjadi 01 Mei 2023
---
Halo Semua,

Ini adalah pengumuman bahwa kedepan akan ada versi berbayar dan perubahan quota dari harian menjadi bulanan dari API.KirimWA.id.

Sebelumnya saya ucapkan terima kasih telah menggunakan API.KirimWA.id. Saat ini jumlah pengguna sudah mencapai 1600+ dan total pengiriman pesan hampir mencapai 1 juta. Sebuah pencapaian yang luar biasa bagi saya.

Akhir-akhir ini mungkin banyak dari pengguna mengeluh kenapa layanan sering mengembalikan HTTP error 503. Hal ini tidak lain karena server sudah tidak mampu menampung jumlah pengguna sebanyak itu. Saya tidak bisa melakukan horizontal scaling untuk mengatasi masalah itu karena arsitektur aplikasi bersifat stateful.

Agar layanan tetap optimal maka mulai awal 10 April 2023 API.KirimWA.id akan menyediakan versi berbayar dan quota akan dihitung **per bulan** bukan per hari. Berikut ini adalah tabel untuk paket layanan bulanan mulai 10 April 2023*.

|Fitur|Gratis|Pro10K|Pro50K|Enterprise|
|--- |--- |--- |--- |--- |
|Harga per bulan|Rp0|Rp49.000|Rp89.000|Rp350.000|
|Quota pengiriman pesan (bulanan) |100|10.000|50.000|Unlimited|
|Pesan tipe teks|✓|✓|✓|✓|
|Pesan tipe gambar|✓|✓|✓|✓|
|Pesan tipe document (semua tipe)|✓|✓|✓|✓|
|Kirim pesan terjadwal|✓|✓|✓|✓|
|Kirim pesan ke group|✓|✓|✓|✓|
|Kirim pesan secara batch|✓|✓|✓|✓|
|~Kirim pesan interaktif~|~✘~|~✓~|~✓~|~✓~|
|Maks. pesan per batch|5|25|50|500|
|Jumlah device per akun|1|2|5|10|
|Mention pengguna pada group|✓|✓|✓|✓|
|Webhook pesan masuk|✓|✓|✓|✓|
|Maks. webhook pesan masuk (diproses)|1.000|25.000|Unlimited|Unlimited|
|Webhook status pengiriman pesan|✓|✓|✓|✓|
|Webhook status perangkat|✓|✓|✓|✓|
|List kontak pada group|✓|✓|✓|✓|
|HTTP request rate limit|30/menit|120/menit|300/menit|Unlimited|
|Dedicated server|✘|✘|✘|✓|
|Dedicated storage|✘|✘|✘|✓|
|Dedicated database|✘|✘|✘|✓|
|~White label (tanpa branding API.KirimWA.id)~|~✘~|~✓~|~✓~|~✓~|
|Support via Email|✘|✓|✓|✓|
|Garansi SLA Uptime |99%|99.5%|99.5%|99.9%|

_*) Fitur dan paket bulanan diatas bisa berubah sewaktu-sewaktu._

### Apa yang akan terjadi pada quota dari akun free saya?

Semua quota akun free akan direset sesuai tabel diatas. Contoh untuk quota akan direset ke 100 per bulan.

### Apa yang akan terjadi pada quota akun berbayar saya setelah 10 April 2023?

Tidak ada perubahan yang terjadi pada akun anda. Tapi anda bisa request untuk pindah ke paket tertentu.

## 10 pemenang mendapatkan gratis paket berbayar selama 1 tahun

Bagi anda yang ingin mendapatkan gratis paket berbayar selama 1 tahun silahkan mengisi feedback form yang saya kirim via email.

10 orang yang beruntung akan mendapatkan paket berbayar gratis selama 1 tahun.

Sekian pengumuman tentang paket berbayar dan perubahan quota. Jika ada yang kurang jelas silahkan sampaikan lewat komentar atau email.

Salam,
Rio Astamal

**Comment by masbagusstwn** [comment]
Wah akhirnya akan launch juga versi berbayarnya :)

Kalau boleh saran mas rio dari saya pribadi, mungkin untuk maks webhook pesan masuk yg diproses mungkin bisa di-up di angka sekitar ribuan mas rio.. hehe.. karna 100 **pesan masuk webhook perbulan** rasanya terlalu sedikit (sekedar saran :D)

**Reply by rioastamal** [reply]
Terima kasih sarannya, nanti dipertimbangkan. Selain itu gunakan properti `allowed_senders` saat membuat/mengupdate webhook agar tidak semua pesan masuk diproses oleh API.KirimWA.id.

**Comment by Fajar-G** [comment]
untuk yang white label maksudnya gimana ya? apakah setiap pesan dari akun free nanti bakalan ada tulisan API.KirimWA.id ? sama kalau berlangganan langsung 1 tahun ada diskon kah. Terima Kasih.

**Reply by rioastamal** [reply]
Akan ada message tambahan disetiap pesan. Contohnya:

```
--
Sent via developer.kirimwa.id
```

Namun keputusan ini belum final dan masih dapat berubah pada saat peluncuran.

**Comment by zonegoh** [comment]
Yuhuu... yang ditunggu-tunggu sudah hadir, gak kecewa seh sama mas rio, kita pakai versi berbayar sampai sekarang aman tentram. tinggal luncurin fitur Whatsapp interaktif message saja. Ini yang lebih keren lagi. thanks mas rio

**Reply by rioastamal** [reply]
Thank you mas @zonegoh sudah menjadi pelanggan Enterprise pertama :D sejak beberapa bulan lalu. Fitur masih dalam proses. Rencana akan saya luncurkan bersama dengan versi pro keluar.

**Reply by shahib-mirbath** [reply]
mana nih formnya?

**Comment by yefridho** [comment]
Nanti kalau udah berbayar semua... minta support 24 jam. Call center standby 24 ya mas. sama garansi UPTIME 99.5% nya ditepati ya mas Rio.

**Reply by rioastamal** [reply]
Terima kasih. Tentunya nanti ada kebijakan penggunaan yang harus disetujui. Seperti yang tertera diatas versi Pro dan Enterprise akan mendapat support via email. Detailnya akan ditentukan kemudian.

**Reply by dp20bp** [reply]
server saat ini sedang down ya mas @rioastamal 
16 Feb 2023 11:13 WIB

**Reply by rioastamal** [reply]
@dp20bp. Sekarang harusnya udah up kembali.

**Reply by zonegoh** [reply]
yang versi berbayar juga down mas, bisa dibantu

**Reply by rioastamal** [reply]
@zonegoh, mohon maaf atas gangguannya, sedang diinvestigasi.

(Update): Sudah up dan running. Detil insiden dan resolusi dikirim via email.

**Reply by dp20bp** [reply]
mantul, iya sudah kembali Up. terima kasih mas @rioastamal .

**Reply by dp20bp** [reply]
mas @rioastamal , apakah kirimWA.id sedang ada maintenance. Tidak dapat response request saat ini.

**Comment by zonegoh** [comment]
Tq, fast respon nya mantap

**Comment by ekoyuliawan** [comment]
Mas @rioastamal bagaimana caranya upgrade ke yang berbayar? Terima kasih.

**Reply by rioastamal** [reply]
@ekoyuliawan saat ini belum bisa kecuali yang versi Enterprise. Untuk yang versi enterprise bisa langsung kirim email ke saya di rio@kirimwa.id.

**Comment by mdestafadilah** [comment]
Mantap neh mang rio, tapi saingannya banyak juga neh... saya bantu list ya

- https://whapi.id/#pricing
- https://whatsva.id/#pricing

**UPDATED**

- https://www.zenziva.id/harga/

saya sudah pakai diatas, uptime sama semua seperti layanan mas rio ... mantap ... sebetulnya saya juga pake diserver sendiri, pake layanan diluar jaringan server kantor biar notif sangat realtime.

untuk zenziva dipakai ketika nomor telpon bukan WA, only SMS. awalnya kita pakai untuk WA, tapi jadi BOROS, karena dihitung per-kredit.

**Comment by rizqifauzan** [comment]
Mas @rioastamal 
Paket free 
100 pesan per bulan 
terllau sedikit mas, buat beberapa user

Paket Rp 50.000
10.000 pesan per bulan
terlalu mahal mas :(
Karena masih tahap coba coba

bagaimana kalau mas Rio tambahkan yang berbasis kouta pemakaian saja
Kredit 50.000 
bisa dipakai untuk 10.000 pesan ( masuk dan keluar )

Atau paket yang Rp. 10.000
buat 1.000 pesan per bulan

Terimakasih mas Rio

**Reply by mdestafadilah** [reply]
kalo basisnya kredit mah udah ada tuh, zenziva

**Reply by rizqifauzan** [reply]
zenziva untuk official WA
kalau yang un official belum nemu

**Reply by mdestafadilah** [reply]
> zenziva untuk official WA kalau yang un official belum nemu

kurang tau doi official atau pake linked device seperti tools buatang mang rio, selama ini saya pakai, zenziva nomor wa nya random

**Reply by rioastamal** [reply]
Bisa dipertimbangkan kedepan

**Reply by rifkisururi** [reply]
saya juga berharap demikian,
sistem nya bayar per pesan

misal per pesan 10 credit

**Reply by mohroyhannahado** [reply]
Setuju banget, paketan sistem Credit aja bang, ini sangat berguna buat saya yg orderannya masih belum ramai. Jd biar sistem notifikasi tetap jalan, tapi tidak berat juga

**Reply by rukandax** [reply]
setuju nih, antara pake credit, atau emang disediain paket yang lebih murah.. soalnya gratis 100 message itu emang kedikitan, jadi pasti prefer paket berbayar, tapi 10k message itu kebanyakan 😅

**Comment by rioastamal** [comment]
**Update 05 April 2023**
- Paket berlangganan terbaru diundur menjadi 01 Mei 2023

**Comment by mohroyhannahado** [comment]
Sekarang sudah 5 Mei bang. Form registrasi berbayarnya dimana?

**Reply by rioastamal** [reply]
Mohon maaf, ada keterlambatan. Akan segera saya umumkan update terbaru dalam minggu ini.

**Reply by rickyok** [reply]
Maap mas Rio, apa ada update mengenai status update registrasi berbayar nya?

**Comment by yefridho** [comment]
503 Service Temporarily Unavailable. Tolong Dong mas Rio.... sistem kami 24 jam harus pakai WA.... bisa di UP lagi servernya Mas. makasih ya mas

**Reply by mdestafadilah** [reply]
> 503 Service Temporarily Unavailable. Tolong Dong mas Rio.... sistem kami 24 jam harus pakai WA.... bisa di UP lagi servernya Mas. makasih ya mas

japri saya pak, saya bisa bantu, pake service saya.

**Comment by wijaya1988** [comment]
boleh nanti dikirimkan form pembayarannya, terima kasih.

**Comment by sorryjack** [comment]
Mas @rioastamal saya mau beli paket berbayar nya,, caranya bagaimana ya?

**Comment by cristanto99** [comment]
Gimana mas @rioastamal , saya mau upgrade ke paket berbayar nih soalnya yang free udah ga cukup buat kebutuhan 🙏

**Comment by yefridho** [comment]
Mas @rioastamal saya juga mau beli paket berbayar nya,, caranya bagaimana ya?

**Comment by rioastamal** [comment]
@yefridho  @cristanto99  @sorryjack  @wijaya1988  untuk sementara, contact via email dengan menyebutkan paket yang diinginkan. Thanks.

**Reply by bramtintonnugroho** [reply]
@rioastamal Mas Rio, untuk paket per tahun nya ada mas ?

**Reply by akbarism** [reply]
@rioastamal mas rio,  saya sudah kirim email 🙏🙏🙏

**Reply by rioastamal** [reply]
Invoice sudah saya kirimkan

**Reply by akbarism** [reply]
> Invoice sudah saya kirimkan

@rioastamal  mas rio saya ada pertanyaan mengenai invoicenya, mohon maaf harus mensyen disini, karena di email belum bales 🙌

**Comment by daviddprtma** [comment]
Halo kak. kemarin saya sudah pesan bot kirim wa nya melalui gform tetapi smp skrg saya masih belum mendapatkan email utk mendapatkan API keynya. Jadi gimana ya kak?

**Reply by rioastamal** [reply]
@daviddprtma sudah dibalas via email. Thanks.

**Comment by agistarakha** [comment]
Untuk webhook termia pesan bisa terima media gambar, audio, dan video tidak?

**Comment by erickwesz** [comment]
Sudah request untuk versi langganannya ke email mas @rioastamal , ditunggu replynya, trims

**Comment by AnonimBella** [comment]
kalau jumlah device per akunnya > 10 tersedia kah untuk paket berbayarnya? 

Jadi sistem yang sedang saya built berkaitan dengan collection / tagih-tagihan. Jikalau menggunakan WA official ada kekurangan di jml devicenya, karena setiap bulannya kita bakal ganti nomor WA secara massal. 

Di kirimwa ini bisa menyediakan paket untuk jumlah devicenya lebih dari 10 tidak?
Saat ini agent concurrent ada >= 20, berarti bakal ada 20 concurrent device yang aktif.

@rioastamal

**Reply by rioastamal** [reply]
@AnonimBella 10 itu soft-limit, sebenarnya unlimited tapi jika terlalu banyak maka server menjadi tidak stabil. Jadi sebenarnya bisa lebih banyak dari 10 kalau memang diinginkan. Tidak ada paket tambahan. Enterprise sudah mencakup itu.

**Reply by mdestafadilah** [reply]
> @AnonimBella 10 itu soft-limit, sebenarnya unlimited tapi jika terlalu banyak maka server menjadi tidak stabil. Jadi sebenarnya bisa lebih banyak dari 10 kalau memang diinginkan. Tidak ada paket tambahan. Enterprise sudah mencakup itu.

Pak rio, bisa dishare ya harga paketnya skrng ke email saya desta@rspersahabatan.co.id terima kasih pa rio.

**Reply by rioastamal** [reply]
@mdestafadilah, harga paket sesuai dengan tabel yang tertera di atas. Untuk enterprise Rp350.000/bulan.

**Reply by AnonimBella** [reply]
> @AnonimBella 10 itu soft-limit, sebenarnya unlimited tapi jika terlalu banyak maka server menjadi tidak stabil. Jadi sebenarnya bisa lebih banyak dari 10 kalau memang diinginkan. Tidak ada paket tambahan. Enterprise sudah mencakup itu.

Pak Rio, barusan saya email ke rio@kirimwa.id. Klien saya tertarik dengan fitur & paket yang ditawarkan kirimwa.id. 
Namun ada beberapa hal yang perlu ditanyakan langsung, sekiranya bisa request Google Meeting atau sejenisnya tidak ya?

Ohiya, di kirimwa.id ada fitur untuk pengecekan status nomor punya whatsapp atau tidak? Saya cek dokumentasinya, tidak ada fitur tersebut.

**Comment by anargyaid** [comment]
halo mas @rioastamal saya sudah kirim konfirmasi pembayaran dari hari kamis tapi sampai seakrang belum dapet token juga 😭😭😭

**Reply by rioastamal** [reply]
@anargyaid maaf, terlewat. Akan segera saya aktivasi dan due date untuk bulan depan akan otomatis saya perpanjang.

**Comment by andes2912** [comment]
Saran mas @rioastamal, mungkin bisa dibuat dashboard web untuk berlangganan agar lebih mudah
